.github/workflows/checks.yml

```diff
@@ -7,6 +7,7 @@ on:
     branches:
       - v1
       - v2
+      - v3
```

Comment:
Not needed as there is no `v3` branch. There is only `v3` version

Reply:
There is v3 branch, I mentioned it in description of this pr

Reply:
Do you think that we don't need this v3? If so I can remove it

Reply:
Please keep it